extensions/wikia/LookupContribs/SpecialLookupContribs_ajax.php

```diff
@@ -131,12 +131,15 @@ public static function prepareLookupUserData( $activityData, $username ) {
 
 		$rows = [];
 		foreach ( $activityData as $row ) {
+			$discussionActivity = LookupUserPage::fetchDiscussionPostCountAndDate($username, $row['id']);
 			$rows[] = [
 				$row['id'], // wiki Id
 				$row['title'], // wiki title
 				$row['url'], // wiki url
 				$wg->Lang->timeanddate( wfTimestamp( TS_MW, $row['lastedit'] ), true ), // last edited
 				$wg->ContLang->formatNum( $row['edits'] ),
+				$discussionActivity['count'],
+				$discussionActivity['date'],
 				LookupUserPage::getUserData( $username, $row['id'], $row['url'] ), // user rights
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)